src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -68,7 +68,7 @@ private static void open(MCreator mcreator) {
 		JPanel props = new JPanel(new GridLayout(4, 2, 5, 5));
 
 		VTextField name = new VTextField(25);
-		JColor color = new JColor(mcreator, false);
+		JColor color = new JColor(mcreator);

```

Comment:
Is there some particular reason null color is now allowed?

Reply:
It's not, if that parameter isn't specified it sets it to false by default (Line 43 of JColor.java)